api/admin/controller/__init__.py

```diff
@@ -635,7 +677,7 @@ def sign_out(self):
             redirect=url_for("admin_view", _external=True),
             _external=True,
         )
-        return redirect(redirect_url)
+        return SanitizedRedirections.redirect(redirect_url)
```

Comment:
It's kind of unnecessary in the sense that it's the tail wagging the dog: If we use `redirect` directly, we'll trigger a code scanning issue because it thinks we aren't sanitizing. If we always go through `SanitizedRedirections.redirect()`, we can ignore the code scanning issue in the one place that we know we've sanitized.